eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody/Internal.hs

```diff
@@ -176,7 +177,7 @@ data BabbageTxBodyRaw era = BabbageTxBodyRaw
   , btbrAuxDataHash :: !(StrictMaybe TxAuxDataHash)
   , btbrNetworkId :: !(StrictMaybe Network)
   }
-  deriving (Generic)
+  deriving (Generic, Typeable)
```

Comment:
Typeable does not need to be derived. This was recently removed. Looks like a rebase gone bad
```suggestion
  deriving (Generic)
```